google-cloud-clients/google-cloud-storage/src/main/java/com/google/cloud/storage/spi/v1/HttpStorageRpc.java

```diff
@@ -288,8 +288,7 @@ public StorageObject create(
                   new InputStreamContent(storageObject.getContentType(), content));
       insert.getMediaHttpUploader().setDirectUploadEnabled(true);
       Boolean disableGzipContent = Option.IF_DISABLE_GZIP_CONTENT.getBoolean(options);
-      if (disableGzipContent != null)
-        insert.getMediaHttpUploader().setDisableGZipContent(disableGzipContent);
+      if (disableGzipContent != null) insert.setDisableGZipContent(disableGzipContent);
```

Comment:
Please add curly braces around the body of the if statement

Reply:
Fixed